src/mono/netcore/System.Private.CoreLib/src/System/Environment.iOS.cs

```diff
@@ -0,0 +1,115 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.IO;
+using System.Runtime.InteropServices;
+
+namespace System
+{
+    partial class Environment
```

Comment:
Nit, `internal`. https://github.com/dotnet/runtime/blob/master/docs/coding-guidelines/coding-style.md

Not sure why our .editorconfig doesn't contain `dotnet_style_require_accessibility_modifiers`

Reply:
FWIW, lots of things in mono's netcore don't follow these style guidelines. We should get to a point where we're running our analyzers on it.  We do enforce visibility via those analyzers.

Reply:
I'm assuming the absence of `dotnet_style_require_accessibility_modifiers` eg., is because there's some debt to pay off.

Do you think we should ask folks to align style as Mono work proceeds? Or build up debt and do it together? I would rather the first, in general.

Reply:
> I'm assuming the absence of dotnet_style_require_accessibility_modifiers eg., is because there's some debt to pay off.

Hopefully there isn't in coreclr/libraries, as we use StyleCop SA1400 to ensure that.  But we're not running the analyzers on mono as far as I can tell (please someone correct me if I'm wrong).  If there is actually debt that the analyzers are missing, I'd love to understand why.

> Do you think we should ask folks to align style as Mono work proceeds? Or build up debt and do it together? I would rather the first, in general.

I'd like to get to a point where we use the same style in mono/netcore as we do elsewhere.  I'm happy to help with that.  Hopefully it'll be a fairly quick / wrote process via VS's auto-fix tooling.

Reply:
@stephentoub if you can please enable the analyzers for mono .cs sources as well. Mono specific C# code is fairly limited so it should not take long to apply the changes

Reply:
A personal opinion on this specific code style. I dislike the rule to force copying the redundant modifiers to all partial parts. This style is much clearer as it says which partial part is the main one and which one is the partial part.

Reply:
> if you can please enable the analyzers for mono .cs sources as well

Ok

Reply:
https://github.com/dotnet/runtime/pull/34052